addons/account/models/chart_template.py

```diff
@@ -32,14 +32,20 @@ def migrate_tags_on_taxes(cr, registry):
         ('module', 'like', 'l10n_%')
     ])
     tax_template_ids = [x['res_id'] for x in xml_records.sudo().read(['res_id'])]
-    for tax_template in env['account.tax.template'].browse(tax_template_ids):
-        tax_id = env['account.tax'].search([
-            ('name', '=', tax_template.name),
-            ('type_tax_use', '=', tax_template.type_tax_use),
-            ('description', '=', tax_template.description)
-        ])
-        if len(tax_id.ids) == 1:
```

Comment:
The real problem is this condition... nothing to do with the company... Just remove this line should be enough.

Reply:
@qdp-odoo Why do you need to ensure there is only one generated _account.tax_ at this point?

Reply:
I imagine it's simply to avoid crap if templates were badly named at first. e.g: if 3 taxe templates have the same name, type_tax_use and description...

Reply:
@sbidoul Ok then, you can simply remove this line instead of adding a loop on companies because this condition is already satisfied by https://github.com/odoo/odoo/blob/10.0/addons/account/models/account.py#L602